source/extensions/filters/http/ext_proc/processor_state.h

```diff
@@ -72,10 +66,6 @@ class ProcessorState : public Logger::Loggable<Logger::Id::ext_proc> {
     BufferedBodyCallback,
     // Waiting for a "body" response in streaming mode.
     StreamedBodyCallback,
-    // Waiting for a "body" response in streaming mode in the special case
-    // in which the processing mode was changed while there were outstanding
-    // messages sent to the processor.
-    StreamedBodyCallbackFinishing,
```

Comment:
This callback state can be removed since we are going to restrict mode_override to header response only. All the code corresponding to this callback state will be  cleaned up.